docs/manual/docs/cmd/graph/teams/teams-clone.md

```diff
@@ -0,0 +1,49 @@
+# graph teams clone
+
+Creates a clone of a Microsoft Teams team
+
+## Usage
+
+```sh
+graph teams clone [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-i, --teamId <teamId>`|The ID of the Microsoft Teams team to clone 
+`-n, --displayName <displayName>`|The display name for the new Microsoft Teams Team 
+`-m, --mailNickname <mailNickname>`|The mail alias for the new Microsoft Teams Team. Known issue: mailNickname property is currently ignored. 
```

Comment:
@sprider , could you please remove that option from the doc since we would drop it until the graph guys fix it.

For more info see: https://github.com/pnp/office365-cli/pull/939#discussion_r279938818

Reply:
@VelinGeorgiev, This is complete.